perf/stability/alertmanager/prometheusrule.yaml

```diff
@@ -47,6 +52,13 @@ spec:
           expr: >
             sum(irate(pilot_total_xds_rejects{app="istiod"}[5m])) / sum(irate(pilot_xds_pushes{app="istiod"}[5m])) > 0.05
           for: 5m
+        - alert: IstiodAvailabilityLow
+          annotations:
+            summary: 'Istiod AvailabilityLow'
+            description: 'Istiod pods have dropped during the last 5m (current value: *{{ printf "%2.0f%%" $value }}*).'
+          expr: >
+            avg(avg_over_time(up{job="istiod"}[1m])) < 0.5
```

Comment:
can we check for crashes specifically?

Reply:
Added alert to check istiod running status with `kube_pod_container_status_running`